portfolio/src/main/webapp/script.js

```diff
@@ -55,6 +56,46 @@ function showFrame() {
 }
 
 function getComments() {
+  myFetch();
+}
+
+async function myFetch() {
```

Comment:
Does this need to be a different function from `getComments()`? Perhaps it does because it's `async`; if so, you could call it something like `getCommentsAsync()`